### PR TITLE
Update process of writing chapters

### LINKED ISSUE #128: Consider how to improve ways of working so that PRs and issues don't get super long
### Detailed description
 - The problem with writing collaboratively and openly with lots of feedback is that some of the threads get SUPER long! See @r-j-arnold's PR for the version control chapter #88 🙀 
 - There are probably some human ways of splitting up these conversations into more manageable chunks....this issue is to capture thoughts around developing that process 👾

### PR DESCRIPTION
**Pull request by r-j-arnold**
In the contributing guidelines update description of the process of writing a chapter and add more detail. Most importantly outline that major and minor reviews of new chapters are to be discussed on separate pull requests.

Following this we can close #128.

- [x] all contributors to this pull request who wish to be included are named in [humans.md](humans.md).

### CODE REVIEW

**Comment by r-j-arnold** [comment]
Following this we can close #128.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -106,13 +106,25 @@ If you feel that we should try new tools or some aspects of the collaboration co
...
+- Copy the chapter template in the templates directory into the markdown file, and commit.
+- Make a pull request to the turing way version of the chapter branch. The title of this request should have the form "[WIP] Write Chapter_name chapter". WIP indicates the chapter is a Work In Progress and not yet ready for review.
 - On your branch add material to the chapter and commit. The goal of this project is to collate and build on the many good resources already available about good practise in data science. As such this material should primarily be drawn from outside sources. Note the link and (if available) license of the source.
 - Once a significant amount of material has been amassed, work (preferably with others) to develop a chapter outline.
-- Once the chapter is reasonably complete, merge a version into master with a note on top of the file that the chapter is work in progress, linking to issues discussing planned changes and additions and - if available - an existing [WIP] follow up pull request. 
 - Edit the amased material into a coherent chapter, adding more material if gaps become apparent.
```
Comment (pherterich) [inline CONTRIBUTING.md]: amased -> amassed

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -106,13 +106,25 @@ If you feel that we should try new tools or some aspects of the collaboration co
...
+- Add a comment on the pull request indicating that this chapter is ready for high level review, i.e discussion of changes of scale of a paragraph or larger such as adding material and restructuring sections.
+- Discuss and make these high level changes on this pull request. Once this is complete merge the chapter into the alan turing intitute's version of the chapter branch.
+- Make another pull request from your fork's version of the branch to the alan turing institute's version of the branch. Title this "[Redy for review] Chapter_name chapter- low level reviews".
+- Discuss and make low level changes to the chapter on this pull request, such as rewording sentences, typos and the like.
+- This division of the pull requests into high and low level changes stops discussion threads becoming unmanagably long.
+- Once this is complete merge the pull request into the alan turing intitute's version of the chapter branch.
+- Merge the alan turing intitute's version of the chapter branch into the alan turing master branch. DO not delete the brnach as the chapter may continue to undergo improvement and development in the future.
```
Comment (pherterich) [inline CONTRIBUTING.md]: brnach -> branch

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -106,13 +106,25 @@ If you feel that we should try new tools or some aspects of the collaboration co
...
 - Once a significant amount of material has been amassed, work (preferably with others) to develop a chapter outline.
-- Once the chapter is reasonably complete, merge a version into master with a note on top of the file that the chapter is work in progress, linking to issues discussing planned changes and additions and - if available - an existing [WIP] follow up pull request. 
 - Edit the amased material into a coherent chapter, adding more material if gaps become apparent.
 - Edit the chapter for style.
+- Once the first draft of the chapter is complete change [WIP] in the pull request title to [Ready for review].
+- Add a comment on the pull request indicating that this chapter is ready for high level review, i.e discussion of changes of scale of a paragraph or larger such as adding material and restructuring sections.
+- Discuss and make these high level changes on this pull request. Once this is complete merge the chapter into the alan turing intitute's version of the chapter branch.
+- Make another pull request from your fork's version of the branch to the alan turing institute's version of the branch. Title this "[Redy for review] Chapter_name chapter- low level reviews".
```
Comment (pherterich) [inline CONTRIBUTING.md]: [Redy for review] -> [Ready for review]

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -106,13 +106,25 @@ If you feel that we should try new tools or some aspects of the collaboration co
...
 ## The process of writing chapters
 
-- Create a branch and create a markdown file on it
-- Make a pull request to the turning way master branch. The title of this request should have [WIP] meaning work in progress at the start.
+- Fork the repository form the alan turing version if you have not done so already.
+- On the alan turing version create a branch with the same name as the chapter to be written.
+- On your fork create a branch with the same name and create a markdown file on it.
+- Copy the chapter template in the templates directory into the markdown file, and commit.
+- Make a pull request to the turing way version of the chapter branch. The title of this request should have the form "[WIP] Write Chapter_name chapter". WIP indicates the chapter is a Work In Progress and not yet ready for review.
 - On your branch add material to the chapter and commit. The goal of this project is to collate and build on the many good resources already available about good practise in data science. As such this material should primarily be drawn from outside sources. Note the link and (if available) license of the source.
```
Comment (pherterich) [inline CONTRIBUTING.md]: While your editing the file, could you quickly go through the whole file and apply the one sentence per line rule please?

**Comment by r-j-arnold** [comment]
@pherterich Nice catches, all addressed, going to merge. Not sure about the issue you mentioned you were also having on #175, I'll look into it tomorrow.

**Pull request merged**